src/components/lists/items/ImportLogListItem.scss

```diff
@@ -0,0 +1,61 @@
+.ImportLogListItem {
+    position: relative;
+
+    &::after {
+        content: "";
+        display: block;
+        clear: both;
+    }
+
+    span {
+        font-size: 1.4em;
```

Comment:
I think it would be nicer to use the default font-size. No other list has big labels like this.